tethys_portal/settings.py

```diff
@@ -392,3 +395,37 @@
 # Add any additional specified settings to module
 for setting, value in portal_config_settings.items():
     setattr(this_module, setting, value)
+
+# Cookie Settings
+# CSRF_COOKIE_SAMESITE = 'Lax'
+# SESSION_COOKIE_SAMESITE = 'Lax'
+
+# Prevent JavaScript from accessing CSRF and session cookies
+# NOTE: This will probably break scripts that use the cookie to get the CSRF token
+# CSRF_COOKIE_HTTPONLY = True
+# SESSION_COOKIE_HTTPONLY = True
+
+# Production only
+# CSRF_COOKIE_SECURE = not DEBUG
+# SESSION_COOKIE_SECURE = not DEBUG
+
+COOKIE_CONFIG = portal_config_settings.pop('COOKIE_CONFIG', {})
+for setting, value in COOKIE_CONFIG.items():
+    setattr(this_module, setting, value)
+
+# CORS Header Settings
+CORS_ALLOWED_ORIGINS = [
```

Comment:
Does this need to be overridden in production?

Reply:
Probably, though I just tested it without these settings and it seems to work still, so I removed them. We need to create a production configuration tutorial eventually for portals with React apps and recommendations for the CORS settings. See #836 I don't think we need it now for the experimental support.